docs/organizational-structure.rst

```diff
@@ -45,7 +45,7 @@ Role Contributors
 -----------------
 
 In context of Ansible roles, each role not directly created by DebOps
-Developers needs a Contributor. The role Contributor signs the role using
+Developers is a Contributor. The role Contributor signs the role using
```

Comment:
Not sure about this. "In context of Ansible roles, each role not directly created by DebOps Developers is a Contributor."

Perhaps it should be "should be authorized by a Contributor"?